index.html

```diff
@@ -17,36 +17,44 @@
             <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
         </td>
       </tr>
-      <td>
-             <div class="container">
-                  <div class="row">
-                      <div class="col-md-4 col-md-offset-3">
-                              <div class="form-group has-feedback">
-                              <label for="search" class="sr-only">Search</label>
-                              <input type="text" class="form-control" name="search" id="search" placeholder="search" value="">
-                              <span class="glyphicon glyphicon-search form-control-feedback"></span>
-                            </div>
-                      </div>
-                  </div>
-              </div>
-        </td>
+
       <tr class="spaceUnder" align="center">
   			<td>
           <button type="button" class="btn btn-success" id="save_now" tabindex='-1'>Save Page Now</button>
           <BR>
-    		</td>
+        </td>
       </tr>
       <tr class="spaceUnder" align="center">
-    		<td>
+        <td>
           <button type="button" class="btn btn-primary" id="recent_capture" tabindex='-1'>Recent Version</button>
           <BR>
-    		</td>
+        </td>
       </tr>
       <tr class="spaceUnder" align="center">
         <td>
           <button type="button" class="btn btn-warning" id="first_capture" tabindex='-1'>First Version</button>
         </td>
-    	</tr>
+      </tr>
+      <tr class="spaceUnder" align="center">
+        <td>
+          <form>
+            <div class="form-group">
+              <label for="tweet_URL_input">Enter URL to be searched in Tweets: </label>
+              <input type="text" class="form-control" id="tweet_URL_input" placeholder="Enter URL to be searched on Twitter" value="archive.org">
```

Comment:
Sorry for the rookie error, did the mistake while resolving the merge conflicts.